tree/dataframe/inc/ROOT/RDF/RSampleInfo.hxx

```diff
@@ -25,7 +25,7 @@ namespace ROOT {
 namespace RDF {
 
 /// This type represents a sample identifier, to be used in conjunction with RDataFrame features such as
-/// DefinePerSample() and per-sample callbacks.
+/// \ref ROOT::RDF::RInterface< Proxied, DS_t >::DefinePerSample "DefinePerSample()" and per-sample callbacks.
```

Comment:
Not a suggestion but I wonder why this doesn't automatically gets turned into a reference but `AsString()` in the line below does 🤔

Reply:
AsString() is a part of this class so I think that's why you don't need anything extra